Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -30,11 +38,76 @@ public static class TextTable
         /// <summary>
         /// Build a table for given message with given padding.
         /// Padding means how many spaces will a message be wrapped with.
-        /// Table itself is made of: "+-" symbols. 
+        /// Table itself is made of: "+-" symbols.
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (string.IsNullOrEmpty(message))
+            {
+                return "";
+            }
+
+            var longestWordInMessageSize = LongestWordInMessageSize(message);
+
+            var sb = new StringBuilder();
+
+            // create a padded line
+            var emptyLine = new StringBuilder();
+            emptyLine.Append(verticalBorderChar).Append(' ', longestWordInMessageSize + (padding * 2)).Append(verticalBorderChar).Append(Environment.NewLine);
+
+            // create horizontal border
+            var horizontalBorder = new StringBuilder();
+            horizontalBorder.Append(cornerChar).Append(horizontalBorderChar, longestWordInMessageSize + (padding * 2)).Append(cornerChar).Append(Environment.NewLine);
+
+            // append top horizontal border
+            sb.Append(horizontalBorder);
+
+            // append padded line
+            for (var i = 0; i < padding; i++)
+            {
+                sb.Append(emptyLine);
+            }
+
+            var wordsInMessage = message.Split(divider);
+
+            // creating and printing text message
+            foreach (var word in wordsInMessage)
+            {
+                sb.Append(verticalBorderChar);
+                sb.Append(' ', padding);
+                sb.Append(word);
+                sb.Append(' ', longestWordInMessageSize - word.Length);
+                sb.Append(' ', padding);
+                sb.Append(verticalBorderChar);
+                sb.Append(Environment.NewLine);
+            }
+
+            // printing padded line
+            for (var i = 0; i < padding; i++)
+            {
+                sb.Append(emptyLine);
+            }
+            // printing bottom horizontal border
+            sb.Append(horizontalBorder);
+
+            return sb.ToString();
+        }
+
+        ///<summary>
+        ///Helper method for finding longest word in message.
+        ///This helps printing appropriate size text, border and padded lines.
+        ///</summary>
+        private static int LongestWordInMessageSize(string message)
```

Comment:
The way you wrote function name sounds like a noun. FindLongestWord is a shorter, more clear name.